homeassistant/components/tuya/select.py

```diff
@@ -135,6 +144,13 @@
             device_class=TuyaDeviceClass.MOTION_SENSITIVITY,
             entity_category=EntityCategory.CONFIG,
         ),
+        SelectEntityDescription(
+            key=DPCode.CHIME_RING_TUNE,
+            name="Chime Ring Tune",
```

Comment:
New entity naming styles requires to use of sentence style casing everywhere.

```suggestion
            name="Chime ring tune",
```